ui/src/Controlpanel.tsx

```diff
@@ -16,7 +16,14 @@ export const ControlPanel = ({ settings, setSettings, reset }: { settings: Setti
             getIndices(),
             getEmbeddingModels(),
         ])
-        const newIndicies = Object.keys(getIndicesResponse).filter((key) => !key.startsWith("."))
+        const newIndiciesMaybeWFeedback = Object.keys(getIndicesResponse).filter((key) => !key.startsWith("."))
+        var newIndicies;
+        if(newIndiciesMaybeWFeedback.includes(FEEDBACK_INDEX_NAME)) {
+            newIndicies = newIndiciesMaybeWFeedback.filter((name) => name !== FEEDBACK_INDEX_NAME)
+        } else {
+            newIndicies = newIndiciesMaybeWFeedback
+            createFeedbackIndex()
```

Comment:
I think this is okay for now but we'll want to move this to the env. setup later

Reply:
agreed. I figure this gives us some flexibility while we figure out if this is a useful feature / what to change about it